src/backend-for-frontend/main.ts

```diff
@@ -30,6 +31,8 @@ async function bootstrap(): Promise<void> {
     const frontendConfig: FrontendConfig = configService.getOrThrow<FrontendConfig>('FRONTEND');
     const redisConfig: RedisConfig = configService.getOrThrow<RedisConfig>('REDIS');
 
+    app.set('trust proxy', frontendConfig.TRUST_PROXY);
```

Comment:
This is optional in the config, but needs to be set here. Is setting it to undefined acceptable?

Reply:
Didn't pose a problem when undefined, but the docs do not clearly state it. Added an undefined-check